CN-Exec: Compile warning for test seed

A file introduced in commit b7099d8f2c causes a C compilation warning  during a clean build of CN:

```
src/cn-testing/test.c: In function ‘cn_test_main’:
src/cn-testing/test.c:66:31: warning: format ‘%llx’ expects argument of type ‘long long unsigned int’, but argument 2 has type ‘uint64_t’ {aka ‘long unsigned int’} [-Wformat=]
   66 |     printf("Using seed: %016llx\n", seed);
      |                         ~~~~~~^     ~~~~
      |                               |     |
      |                               |     uint64_t {aka long unsigned int}
      |                               long long unsigned int
      |                         %016lx
```

--- a/runtime/libcn/src/cn-testing/test.c
+++ b/runtime/libcn/src/cn-testing/test.c
@@ -4,6 +4,7 @@
 #include <string.h>
 #include <setjmp.h>
 #include <signal.h>
+#include <inttypes.h>
 
 #include <cn-executable/utils.h>
 
@@ -63,7 +64,7 @@ int cn_test_main(int argc, char* argv[]) {
         printf("Running in interactive mode\n");
     }
 
-    printf("Using seed: %016llx\n", seed);
+    printf("Using seed: %016" PRIx64 "\n", seed);
     cn_gen_srand(seed);
     cn_gen_rand(); // Junk to get something to make a checkpoint from
 

